tests/sweep_runner/test_param_sweep_runner.py

```diff
@@ -115,20 +120,90 @@ def tests_prepare_raises_error_if_too_many_sino_slices(mocker: MockerFixture):
         chunk_shape=GLOBAL_SHAPE,
         block_start=0,
     )
-    loader = make_test_loader(mocker, block=block)
-    m1 = make_test_method(mocker)
-    sweep_wrapper = make_test_method(mocker, param_1=(0,))
-    pipeline = Pipeline(loader=loader, methods=[m1, sweep_wrapper])
-    runner = ParamSweepRunner(pipeline, MPI.COMM_WORLD)
+    preview = PreviewConfig(
+        angles=PreviewDimConfig(start=0, stop=180),
+        detector_y=PreviewDimConfig(start=0, stop=100),
+        detector_x=PreviewDimConfig(start=0, stop=160),
+    )
 
-    with pytest.raises(ValueError) as e:
-        runner.prepare()
+    class FakeModule:
+        def paganin(data: np.ndarray, alpha: float, pixel_size: float, energy: float, dist: float):  # type: ignore
+            return data * alpha
 
-    err_str = (
-        "Parameter sweep runs support input data containing <= 7 sinogram slices, "
-        "input data contains 10 slices"
+    mocker.patch(
+        "httomo.method_wrappers.generic.import_module", return_value=FakeModule
     )
-    assert err_str in str(e)
+
+    # Stuff take from `make_test_loader()`
```

Comment:
Sorry, when I put this comment in the snippet I sent previously, I added it to try and help you to understand the code snippet, I don't think this needs to be in the final test.